simphony/core/cuba.yml

```diff
@@ -474,3 +474,52 @@
   number: 67
   shape: [1]
   type: double
+- description: Point item identifier
+  domain: [ATM, DEM, FEM, FVM, LBM, SPH, VIS]
```

Comment:
Please restrict the domain for the new attributes. If I am correct they are not valid for all.